gradle.properties

```diff
@@ -38,7 +38,7 @@ POM_SCM_CONNECTION=scm:git:git://github.com/slackhq/slack-lints.git
 POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/slackhq/slack-lints.git
 POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=slackhq
-POM_DEVELOPER_NAME=Slack Technologies, Inc.
+POM_DEVELOPER_NAME=Slack Technologies, LLC.
 POM_DEVELOPER_URL=https://github.com/slackhq
-# Release signing isn't necessary as this isn't publicly distributed
-RELEASE_SIGNING_ENABLED=false
+POM_INCEPTION_YEAR=2021
+VERSION_NAME=1.0.0-SNAPSHOT
```

Comment:
Should this be something more specific? `1.0.0-SNAPSHOT` is the default as I recall